tests/cover_crypt.non_regression_vector.ts

```diff
@@ -255,37 +255,37 @@ export class NonRegressionVector {
     this.lowSecretFinTestVector.decrypt(this.topSecretFinKey.key)
     try {
       this.lowSecretMkgTestVector.decrypt(this.topSecretFinKey.key)
-      throw new Error("Should be able to decrypt")
+      throw new Error("Should not be able to decrypt")
```

Comment:
After doing the same in Python, I realized that the thrown error will be automatically caught by the catch clause below, not showing the error message so I decided to use a simple `print` in python.